pyproject.toml

```diff
@@ -79,6 +79,7 @@ dependencies = [
     "parsedatetime",
     "paramiko>=3.4.0",
     "pgsanity",
+    "Pillow==11.3.0",
```

Comment:
Is there some reason we're pinning to 11.3.0, and not a range/lower bound?

Reply:
We can use range >=11, <12. We should use at least 11, as in versions <=10 they used a different license

Reply:
I think `>=11, <12` sounds good. I think there was some discussion a while back to try to avoid having upper bounds, as it restricts dependabot. So maybe just `>=11.3.0` is ok (people should be using lock files in prod anyway..)